README.md

```diff
@@ -26,6 +26,7 @@ This Action commits the contents of your Git tag to the WordPress.org plugin rep
 
 ### Inputs
 * `generate-zip` - Generate a ZIP file from the SVN `trunk` directory. Outputs a `zip-path` variable for use in further workflow steps. Defaults to false.
+* `commit-message` -  Use git last commit message if true, or "Update to version $VERSION from GitHub" if false else any text. Defaults to false.
```

Comment:
Yes, this is better. Though some would still want to use resolved text. For example I currently use the below code to extract the version to supply to the github action 

```yml
    - name: Extract Version
      run:
        echo "STABLE_TAG=$(find . -iname README.TXT -exec grep -oiP -m 1 'stable\s+tag\s*:\s\K.*' {} \;)" >> $GITHUB_ENV
```
then
```yml
    - name: WordPress Plugin Deploy
      id: deploy
      uses: 10up/action-wordpress-plugin-deploy@stable
      env:
        SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
        SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
        SLUG: ...
        VERSION: ${{ env.STABLE_TAG }}
        BUILD_DIR: build
```

A table would be able to convey the message better.

<table>
    <thead>
        <tr>
            <th>Input</th>
            <th>Description</th>
            <th>Default</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td ><code>use-commit-message</code></td>
            <td><span>Accepts either a boolean or text value</span>
<ul>
<li><code>true</code>&nbsp;:&nbsp;<span>Will use the last Git Commit message.</span></li>
<li><code>false</code>&nbsp;:&nbsp;<span>Will use "Update to version $VERSION from GitHub"</span></li>
<li><code>text</code>&nbsp;:&nbsp;<span>Any text value to be used as the commit message. Useful for when you want to dynamically generate the commit message.</span></li>
</ul>
</td>
 <td><code>false</code></td>
        </tr>
        <tr>
            <td><code>generate-zip</code></td>
            <td>...</td>
            <td>...</td>
        </tr>
    </tbody>
</table>

Or

#### Inputs

1. `use-commit-message` : Accepts either a boolean or text value. default `false`
       - `true` : Will use the last Git Commit message.
       - `false` : Will use "Update to version $VERSION from GitHub"
       - `text` : Any text value to be used as the commit message. Useful for when you want to dynamically generate the commit message.
 
2. `generate-zip` : ...

So if documented properly it can be made clear what inputs it takes, what it defaults to, etc. I was going to do a PR for the documentation but ran out of time.

You might also want to remove the extra git from `Use git the latest git...`